ocp_resources/resource.py

```diff
@@ -822,6 +822,25 @@ def labels(self):
         """
         return self.instance["metadata"]["labels"]
 
+    def conditions_sampler(self, timeout):
```

Comment:
Gets the resource not specifically `conditions` - should either be renamed or change what is yielded